problemsets-mat223/modules/module10-exercises.tex

```diff
@@ -1,17 +1,47 @@
 \begin{exercises}
+    %Patrick:
+        %45 min
+        %60 min
+        %10 min
+        %10 min
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob \label{PROB-MOD10-Q1}Let $\mathcal S:\R^2\to\R^2$ be a transformation that doubles every vector, and let $M_{\mathcal S}$ be the corresponding matrix. Find $M_{\mathcal S}$.
+	    \prob \label{PROB-MOD10-Q2}Let $\mathcal R:\R^2\to\R^2$ be rotation clockwise by $135^\circ$, and let $M_{\mathcal R}$ be the corresponding matrix. Find $M_{\mathcal R}$.
+	    \prob \label{PROB-MOD10-Q3}Let $\mathcal P:\R^2\to\R^2$ be projection onto the $y$-axis,and let $M_{\mathcal P}$ be the corresponding matrix. Find $M_{\mathcal P}$.
+	    \prob \label{PROB-MOD10-Q4}Let $\mathcal F:\R^2\to\R^2$ be a transformation reflecting over the line $y=x$,and let $M_{\mathcal F}$ be the corresponding matrix. Find $M_{\mathcal F}$.
+	    
+	    \prob Let $\mathcal U:\R^2\to\R^2$ be the matrix transformation given by $M=\mat{0&0\\-\sqrt{2}/2 & -\sqrt{2}/2}$, let $\mathcal R$ and $\mathcal P$ be the transformations given in problems \ref{PROB-MOD10-Q1} and \ref{PROB-MOD10-Q2}.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item   Compute the matrix $M_1$ that gives $\mathcal R \circ \mathcal P$.
+			\item   Compute the matrix $M_2$ that gives $\mathcal P \circ \mathcal R$.
+			\item   Write $\mathcal U$ as the composition (in some order) of $\mathcal R$ and $\mathcal P$.
 		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
-
-	\end{problist}
+		
+		\prob Let $\mathcal V:\R^2\to\R^2$ be the matrix transformation given by $M=\mat{0&2\\2&0}$, let $\mathcal F$ and $\mathcal S$ be the transformations given in problems \ref{PROB-MOD10-Q3} and \ref{PROB-MOD10-Q4}.
```

Comment:
It's annoying to have to look to other problems to find the transformations. Just restate them rather than referencing them.